packages/terra-application-utility/src/_Utils.js

```diff
@@ -0,0 +1,196 @@
+};
+
+const TITLES = {
+  MENU: 'Menu',
+  USER_INFORMATION: 'User Information',
+  CHANGE_PHOTO: 'Change Photo',
+  SETTINGS: 'Settings',
+  APPEARANCE: 'Appearance',
+  SECURITY: 'Security',
+  HELP: 'Help',
+  GETTING_STARTED: 'Getting Started',
+  ABOUT: 'About',
+  TERMS_OF_USE: 'Terms of Use',
+  LOG_OUT: 'Log Out',
+};
+
+const defaultConfig = userData =>
+  [
+    {
+      key: KEYS.MENU,
+      title: TITLES.MENU,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [
+        KEYS.USER_INFORMATION,
+        KEYS.SETTINGS,
+        KEYS.HELP,
+        KEYS.LOG_OUT,
+      ],
+    },
+    {
+      key: KEYS.USER_INFORMATION,
+      title: TITLES.USER_INFORMATION,
+      content: userData,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [
+        KEYS.CHANGE_PHOTO,
+      ],
+    },
+    {
+      key: KEYS.SETTINGS,
+      title: TITLES.SETTINGS,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [
+        KEYS.APPEARANCE,
+        KEYS.SECURITY,
+      ],
+    },
+    {
+      key: KEYS.LOG_OUT,
+      title: TITLES.LOG_OUT,
+      contentLocation: LOCATIONS.FOOTER,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.HELP,
+      title: TITLES.HELP,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [
+        KEYS.GETTING_STARTED,
+        KEYS.ABOUT,
+        KEYS.TERMS_OF_USE,
+      ],
+    },
+    {
+      key: KEYS.CHANGE_PHOTO,
+      title: TITLES.CHANGE_PHOTO,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.APPEARANCE,
+      title: TITLES.APPEARANCE,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.SECURITY,
+      title: TITLES.SECURITY,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.GETTING_STARTED,
+      title: TITLES.GETTING_STARTED,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.ABOUT,
+      title: TITLES.ABOUT,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+    {
+      key: KEYS.TERMS_OF_USE,
+      title: TITLES.TERMS_OF_USE,
+      contentLocation: LOCATIONS.BODY,
+      isSelected: false,
+      isSelectable: false,
+      childKeys: [],
+    },
+  ];
+
+const itemShape = PropTypes.shape({
+  /**
+   * The unique key associated with this item.
+   */
+  key: PropTypes.string.isRequired,
+  /**
+   * The component associated with this item.
```

Comment:
Is there expected keys or object structure associated with the `content` object prop?

Reply:
I think this should be node or element, based on how I'm consuming it.